src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -463,7 +463,7 @@ public void SarifLogger_ScrapesFilesFromResult()
             {
                 string fileName = @"file" + i + ".cpp";
                 string fileDataKey = "file:///" + fileName;
-                sarifLog.Runs[0].Files.Where(f => f.FileLocation.Uri.AbsolutePath.Contains(fileDataKey)).Any().Should().BeTrue();
+                sarifLog.Runs[0].Files.Where(f => f.FileLocation.Uri.AbsoluteUri.ToString().Contains(fileDataKey)).Any().Should().BeTrue();
```

Comment:
>AbsoluteUri [](start = 69, length = 11)

This change apparently fixes this test. But in your branch, three other SarifLoggerTests went from failing to passing (see the table I sent you), but I don't see any changes that would account for that.